src/utils/form/getErrorMessage.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+function getErrorMessage(meta: FinalFormMeta, validateType: ?JInputValidateType): ?string {
+  const err = meta.error || meta.submitError
```

Comment:
maybe call it `metaError`?

it's not ok to have 2 vars `err` & `error`

metaError & strError would be great